src/static/welcome.php

```diff
@@ -32,7 +32,7 @@
             </div>
             <div class="rightside">
                 <div class="nav-linkwrapper">
-                    <a href="index.html">Home</a>
```

Comment:
this is another page that can be gotten rid of since we are making them login first.